plugins/mpd/mpdstats_

```diff
@@ -83,7 +91,7 @@ ACTION="$(basename "$0" | sed 's/^.*_//')"
 do_autoconf () {
     if [ -z "$NCBIN" ] ; then
         echo "no (missing netcat program ('nc'))"
-    elif ! echo version | "$NCBIN" "$MPDHOST" "$MPDPORT" >/dev/null 2>&1; then
+    elif ! echo stats | "$NCBIN" "$NCARGS" "$MPDHOST" "$MPDPORT" >/dev/null 2>&1; then
```

Comment:
I guess, you want to access *multiple* arguments for netcat here (or even none at all), thus you may not quote these in the call (they would be a single token and thus rejected by netcat).
In order to work around the resulting shellcheck complain, an override (something like `# shellcheck disable=SC....`) in the line above (or at the beginning of the `if` conditional) would be necessary.

Reply:
Well, actually it works like this with `env.ncargs -w 1` but I unquoted those anyway!

Reply:
Indeed - `nc` seems to handle parameter pairs (here `-w 1`) somehow special. With `-4 -w 1` it breaks (as expected).
Thanks for fixing it!